src/Uno.XamlMerge.Task/FullPathComparer.cs

```diff
@@ -10,8 +10,7 @@ internal class FullPathComparer : IEqualityComparer<ITaskItem>
 
     public bool Equals(ITaskItem x, ITaskItem y)
     {
-        return x.GetMetadata("FullPath") == y.GetMetadata("FullPath")
-           && x.GetMetadata("MergeFile") == y.GetMetadata("MergeFile");
+        return x.GetMetadata("FullPath") == y.GetMetadata("FullPath");
```

Comment:
We don't need the `GetMetadata("MergeFile")` comparison any more?

Reply:
It's not working in the previous PR, so no :) The problem is the comparison at the entry of the Task, for deduplication.